### PR TITLE
Added help text when choosing project space name

### PR DESCRIPTION
**Pull request by orangejenny**
https://trello.com/c/Vekt9Xa5/75-help-text-for-users-creating-new-project-space-that-this-name-is-permanent-jhpiego-test

screen shot 2018-11-06 at 11 50 29 am

@jmtroth0

### CODE REVIEW

**Comment by orangejenny** [comment]
fyi @ohmegasquared

File: corehq/apps/registration/templates/registration/domain_request.html [hunk 1]
```diff
@@ -7,15 +7,20 @@
...
+                {% endblocktrans %}
+            {% else %}
+                {% blocktrans %}
+                    Create a New Project Space
+                {% endblocktrans %}
+            {% endif %}
+        </h1>
+        <div class="help-block">
+            Please create a name for your project.
+            This will be used to create a project URL and will not be able to be changed in the future.
```
Comment (jmtroth0) [inline corehq/apps/registration/templates/registration/domain_request.html]: "and `cannot` be changed in the future" or "and `you` will not be able to change it" ?

**Review by jmtroth0 — APPROVED** [review]

File: corehq/apps/registration/templates/registration/domain_request.html (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/registration/templates/registration/domain_request.html]: updated

**Review by jmtroth0 — APPROVED** [review]
that passive voice

**Comment by ohmegasquared** [comment]
@orangejenny, did you update this with the col widths? Also, do we set a character limit? 

Good language check, @jmtroth0!

**Comment by orangejenny** [comment]
@ohmegasquared I'm going to do the column widths in other PRs, starting with https://github.com/dimagi/commcare-hq/pull/22325  Also not going to do the character limit in this PR, I'll make a note on the trello card.

**Pull request merged**